### DISCUSSION TITLE
Welcome to crab Discussions!

### DISCUSSION THREAD

**Discussion by trvswgnr** [crablang/crab#36] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dogue** [comment]
I like crab.

**Comment by brohamgoham** [comment]
My names mohammed, i am a crab lang enthusiast & want to know where & When can i start publishing packages? i want to help 🔨🧱 build the ecosystem for Crab Lang

**Reply by trvswgnr** [reply]
our primary focus is getting a proper compiler and install script working for crablang. that said, we'd love to showcase some packages from supportive members of the community so it won't be long!

**Comment by Kapendev** [comment]
I think it would be a good (maybe even great) idea to add a garbage collector to crab, making it automatically super safe and noob friendly! But it should not be like the Java one (very slow and bad and probably full of ub). Love crab and hope to see it grow to something life changing.

**Reply by ghost** [reply]
......not having the garbage collector and using the borrow checker is like, the whole point..............

**Reply by Kapendev** [reply]
Hehe. I'm just trolling but I hope things go well for the project.
Here's a more serious question: Is there a roadmap for the project? Or something similar that explains what the community and the devs want to focus on for now?

**Comment by alexhercorp** [comment]
I've noticed little communication since a couple of days ago, is the project still ongoing? ...

**Reply by trvswgnr** [reply]
? the project is still very much ongoing. this thread got lost in the noise is all.

**Comment by P1R** [comment]
When DAO?
